packages/web-pkg/src/composables/actions/types.ts

```diff
@@ -12,6 +12,7 @@ export interface Action<T = ActionOptions> {
   componentType: 'button' | 'router-link'
   class: string
   canBeDefault?: boolean
```

Comment:
Hey @diocas and @elizavetaRa 

my intend (see discussion, I think yours as well at some point) was to rephrase `canBeDefault` to `hasPriority`. Meaning, having both in parallel now makes things unnecessarily complex. 🙈 

On top of that: I think the approach with the app provider - define via config which app has priority - is the way to go. An extension developer should never be the one who decides which application has priority over another one. The admin has to decide that, so that's configuration only, not code.

Could you look into realizing the following?
- get rid of `canBeDefault` entirely
- introduce `hasPriority` but only through configuration, nothing hardcoded. For the app provider apps it's already there with the `default_application` attribute on mime types.
- determine the app that gets opened on click in file list (= "default app") by throwing all standalone apps and all app provider apps into one list, sort them by priority, then by internal vs app provider, and pick the first.

What do you think? Does that make sense?

Reply:
@diocas @elizavetaRa anything we can do to support you here? Do you have feedback or further need for clarification?

Reply:
@elizavetaRa has been a bit busy. I'll check with her tomorrow when she can fit this in her tasks.

Reply:
@diocas if you're generally fine with what Benedikt suggested I could also take over?

Reply:
@pascalwengerter @kulmann I'll take this issue, will update the pr these days, thanks for waiting!

Reply:
@kulmann  2 questions:
1) what is the purpose of "canBeDefault" in terms priority in the composables/actions files like `web-app-files/src/composables/actions/files/useFileActionsCreateNewFile.ts`?
2) "introduce `hasPriority` but only through configuration, nothing hardcoded."  You mean move setting` hasPriority` of all the extensions, also editors to config.json instead of index.ts of each extension?

Reply:
hey @elizavetaRa - cool that you are working on this, thank you! 🤩 
1. in the case of file action composables, `canBeDefault` doesn't belong there. Could be removed entirely, doesn't make sense.
2. yes, that's exactly what I meant. :-) Please note that providing config for an app is not possible in the `apps` array of the config.json, but only for the `external_apps` array. That one is a bit misleading, name wise, but that's fine for now. Only means that if an admin wants to give an app priority over another one for the same file type they need to define the app in `external_apps`. We need to clean that up (big time), but I wouldn't do that in the scope of this PR. So please just keep the assumption that providing config for an app is only possible in the `external_apps` array of the config.json. That's fine.

Reply:
So proposal 2 for config.json:

  "external_apps": [
    ...,
    {
      "id": "A",
      "path": "/aaa",
      "config": {
        "priorityExtensions": ["txt"]
        ]
      }
    }
  ],